scale-testing.mk

```
@@ -38,6 +47,36 @@ endif
 	$(TERRAFORM) -chdir=$(TF_DEPLOY_STATE_DIR) apply -auto-approve || exit 1
 	rm -f $(TF_DEPLOY_STATE_DIR)/*.tfvars
 
+.PHONY: import-aws-vars
+import-aws-vars:
+-include $(TF_EKS_STATE_DIR)/outputs.env
+
+.PHONY: connect-cluster
+connect-cluster: import-aws-vars ## Connect to the EKS cluster
+	aws eks --region $(AWS_REGION) update-kubeconfig --name $(EKS_CLUSTER_NAME)
+
+.PHONY: cleanup-port-forward
+cleanup-port-forward: ## Kill orphan port-forward processes
+	@echo "Cleaning up orphan port-forward processes..."
+	@pgrep -f 'kubectl port-forward -n $(K8S_VAULT_NAMESPACE) statefulset/vault' | xargs -r kill -9 && \
+		echo "Port-forward processes terminated successfully." || \
+		echo "No port-forward processes found or an error occurred."
+
+.PHONY: set image scale-tests
+scale-tests: cleanup-port-forward set-image connect-cluster import-aws-vars
+	$(MAKE) port-forward &
+	SCALE_TESTS=true VAULT_ENTERPRISE=true ENT_TESTS=$(VAULT_ENTERPRISE) \
+	SUPPRESS_TF_OUTPUT=$(SUPPRESS_TF_OUTPUT) SKIP_CLEANUP=$(SKIP_CLEANUP) \
+	OPERATOR_IMAGE_REPO=$(IMAGE_TAG_BASE) OPERATOR_IMAGE_TAG=$(VERSION) \
+	OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE) \
+	VAULT_OIDC_DISC_URL=$(EKS_OIDC_URL) VAULT_OIDC_CA=false \
+	INTEGRATION_TESTS=true EKS_CLUSTER_NAME=$(EKS_CLUSTER_NAME) K8S_CLUSTER_CONTEXT=$(K8S_CLUSTER_CONTEXT) CGO_ENABLED=0 \
```

nit: split long line 73 
```suggestion
	INTEGRATION_TESTS=true EKS_CLUSTER_NAME=$(EKS_CLUSTER_NAME) \
	K8S_CLUSTER_CONTEXT=$(K8S_CLUSTER_CONTEXT) CGO_ENABLED=0 \
```